app/Http/Controllers/SubmissionController.php

```diff
@@ -61,7 +62,11 @@ public function submit(): Response|JsonResponse
     private function failProcessing(?string $filename, int $outResponseCode, string $outMessage): void
     {
         if ($filename !== null) {
-            Storage::move('inbox/' . $filename, 'failed/' . $filename);
+            try {
+                Storage::move('inbox/' . $filename, 'failed/' . $filename);
+            } catch (Exception $e) {
```

Comment:
Would the more specific exception `League\Flysystem\UnableToMoveFile` be more appropriate here?